pkgs/sdk-pkg/src/genie/libs/sdk/apis/iosxe/eigrp/verify.py

```diff
@@ -122,3 +122,62 @@ def verify_eigrp_neighbors(
             return set(neighbors).issubset(eigrp_neighbors)
         log.error(f"Please, provide a valid format for AS, Neighbors, ip and/or vrf")
         continue
+
+
+def verify_eigrp_router_id(
+    device,
+    router_id=None,
+    vrf="default",
+    AS_N=None,
+    ip="ipv4",
+    max_time=60,
+    check_interval=10,
+):
+    """verify EIGRP IDs for a given vrf and AS active instance for ipv4 or ipv6
+    Args:
+      device (obj): Device object
+      router_id = None (list): List of router ID to check
+      vrf = "default" (str): Name of the vrf by default set to "default"
+      AS_N = None (int) : Autonomous System
+      ip = "ipv4" (str): Protocol ip set by default to "ipv4" to change to "ipv6"
```

Comment:
```suggestion
      router_id  (list): List of router ID to check
      vrf  (str): Name of the vrf by default set to "default"
      AS_N (int) : Autonomous System
      ip (str): Protocol ip set by default to "ipv4" to change to "ipv6"
```

Reply:
Hi @lsheikal I think here is the same. there is no change to make!

Reply:
it is a required change, to keep the docstrings consistent.